### ISSUE TITLE
About result

### ISSUE THREAD

**Issue by qq1332427275** [HashWang-null/IRBridge#1]
Hello, author! Excellent work! I tried to replicate your paper's results on low-light image enhancement, but when I only modified the load_data parameter for training, I found that the loss quickly overfitted, and the results were very poor when sampling from saved checkpoints. What's the reason for this? What else should I modify? I look forward to your feedback!
